### DISCUSSION TITLE
Unknown argument "footerOverride" on field "SiteMutation.updateConfig"

### DISCUSSION THREAD

**Discussion by edacval** [requarks/wiki#6479] [Error / Bug Report] [answered]
On navigating /a/general , red notice  ` unexpected error occurred.`  appears for some seconds. When clicking on "APPLY" ,  red notice `Unknown argument "footerOverride" on field "SiteMutation.updateConfig" ` appears and no changes are saved. 

wiki-js version: v2.5.299
Operating System: Arch Linux 
Kernel 6.1.31-1-lts x64
Node.js: 16.20.0
PostgreSQL: 15.3

**Comment by NGPixel** [comment, marked as answer]
Try clearing your browser cache?

**Reply by edacval** [reply]
Thanks, solved.
